tests/OpenTok/ArchiveTest.php

```diff
@@ -103,9 +105,10 @@ public function testStopsArchive()
         $this->assertNotEmpty($contentType);
         $this->assertEquals('application/json', $contentType);
 
-        $authString = $request->getHeader('X-TB-PARTNER-AUTH');
+        $authString = $request->getHeader('X-OPENTOK-AUTH');
         $this->assertNotEmpty($authString);
-        $this->assertEquals($this->API_KEY.':'.$this->API_SECRET, $authString);
+        $decodedToken = JWT::decode($authString, $this->API_SECRET, array('HS256'));
+        $this->assertEquals($this->API_KEY, $decodedToken->iss);
```

Comment:
this comment pretty much goes in all the tests, but can we include a few more assertions? see here: https://github.com/opentok/Opentok-Python-SDK/blob/613e5368748ec31a18c4f47c26683bb3bcdbbcf1/tests/validate_jwt.py. you might want to throw this in `TestHelpers.php`